blockapps-strato/README.md

```diff
@@ -0,0 +1,37 @@
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
```

Comment:
This deploy button code is the v1. Since then we’ve moved to official Portal. You need something like below code for the deployment to go to Portal. URL Encode the raw url to the azuredeploy.json file and put it after https://portal.azure.com/#create/Microsoft.Template/uri/.

[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-application-gateway-public-ip-ssl-offload%2Fazuredeploy.json)

The above link is for this template https://github.com/Azure/azure-quickstart-templates/tree/master/101-application-gateway-public-ip-ssl-offload